packages/iframe-stamper/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # @turnkey/iframe-stamper
 
+## 1.2.0
+
+### Minor Changes
+
+- 0281b88: Remove optional publicKey parameter from injectKeyExportBundle.
+- 0e3584a: Add optional keyFormat and publicKey parameters to injectKeyExportBundle. Add extractKeyEncryptedBundle.
```

Comment:
just to confirm, the change here is that `publicKey` is no longer a parameter? should we update this changelog with the _net_ changes? not a blocker regardless